spec/infinispan_cluster_spec.js

```diff
@@ -30,11 +33,467 @@ describe('Infinispan cluster client', function() {
       .catch(t.failed(done)).finally(done);
   });
 
+  it('can put a key/value pair using different clients', function(done) {
```

Comment:
Hmmmm, I see - I haven't taken it into account. 
Yes, you are completely right - that was the purpose of these tests. Then I guess the easiest way would be to run these tests on replicated cache or maybe enable the L1 on distributed cache. What do you think? 

For the refactoring - great idea! I will change that.